src/segment-loader.js

```diff
@@ -1371,6 +1371,8 @@ export default class SegmentLoader extends videojs.EventTarget {
 
     const Cue = window.WebKitDataCue || window.VTTCue;
     const value = {
+      dateTimeObject: (segment && segment.dateTimeObject) || null,
+      dateTimeString: (segment && segment.dateTimeString) || null,
```

Comment:
this should probably default to the empty string.

Reply:
Didn't seem accurate to me. I felt null was a better representation of the value not being there. I can change if `''` is preferred, though.

Reply:
I think it should either be a string to match the type of that property when a value is present, or, maybe better yet, not have the property at all if a value is absent.

Reply:
I disagree; but ok :) I'll make the update.

Reply:
Why do you think having null values would be better?

Reply:
so you're ok with dateTimeObject being null? You just want dateTimeString to be `''`?

or are we sticking with removing them all together when not present?

Reply:
@rart I believe what you have is fine, if the program date time tag has no value for some reason (not sure if this is allowed in the HLS spec) I don't think we should have the property be present with an empty string but rather use null.